qdax/environments/__init__.py

```diff
@@ -148,6 +151,19 @@ def create(
     if eval_metrics:
         env = brax.envs.wrappers.EvalWrapper(env)
         env = CompletedEvalWrapper(env)
+    if fixed_init_state:
+
+        base_env_name_fixed_state_wrapper: str
+        if env_name in _qdax_custom_envs.keys():
+            base_env_name_fixed_state_wrapper = typing.cast(
```

Comment:
What is that for?

Reply:
it is just to avoid issues with the type checker. It was either this or "type: ignore".
it does not change the value of the variable.

Reply:
I think I personally prefer the "type: ignore" haha